data/mods/ssb/abilities.ts

```diff
@@ -10,49 +99,287 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 	},
 	*/
 	// Please keep abilites organized alphabetically based on staff member name!
-	// 5gen
-	seasonsgift: {
-		desc: "If Sunny Day is active, this Pokemon's Attack is 1.5x and its Speed is doubled.",
-		shortDesc: "If Sunny Day is active, this Pokemon's Attack is 1.5x and its Speed is doubled.",
-		name: "Season's Gift",
-		isNonstandard: "Custom",
-		onModifyAtk(atk) {
-			if (this.field.isWeather(['sunnyday', 'desolateland'])) {
-				return this.chainModify(1.5);
+	// Aelita
+	scyphozoa: {
+		desc: "On switch-in, this Pokemon removes all field conditions, entry hazards, and stat boosts on both sides, gaining one random boost for every field condition, entry hazard, or boosted stat that gets cleared. This Pokemon's moves ignore abilities. If this Pokemon is a Zygarde in its 10% or 50% Forme, it changes to Complete Forme when it has 1/2 or less of its maximum HP at the end of the turn.",
+		shortDesc: "Power Construct + Mold Breaker. On switch-in, clears everything for random boosts.",
+		name: "Scyphozoa",
+		onSwitchIn(source) {
+			const stats: BoostName[] = [];
+			let stat: BoostName;
```

Comment:
You shouldn't need to `let` this; the for loop initializes it for you!

Reply:
it doesn't recognize the type if you initialize it within the loop and gives a typeerror.

Reply:
Oh, that's.... really fucking weird. Thanks, TypeScript!